app/src/main/java/nl/tudelft/b_b_w/model/GetDatabaseHandler.java

```diff
@@ -48,11 +55,43 @@ public final int lastSeqNumberOfChain(String owner) {
     }
 
     /**
+     * Checks wheter a given key is already revoked
+     * @param owner the owner (NOT user) of the key
+     * @param key the key in question
+     * @return
+     */
+    public final boolean containsRevoke(String owner, String key) {
+        List<Block> blocks = getAllBlocks(owner);
+        for (Block block : blocks) {
+            if (block.getPublicKey().equals(key) && block.isRevoked())
+                return true;
+        }
+        return false;
+    }
+
+    public final boolean containsRevoke_fasterButNotWorking(String owner, String key) {
```

Comment:
This is not a good method name, you may not give it an underscore. Next to that, the method name is too long.